### PR TITLE
Creating a limited JS file, removing file upload

### PR DESCRIPTION
**Pull request by moflo**
Hi, nice library! We needed to build a reduced version for mobile users and so wanted to remove the file upload form element option. Wish there was a more elegant way of doing this, but we wound up selectively eliminating some coffee scripts & views. Also, we needed to create a build which reused some of the more common JS libraries and so created a new `grunt` build flow to produce `vendor_limited.js` which is used by the matching `index_limited.html` file.

Cheers!

### CODE REVIEW

**Comment by al3x** [comment]
Would you mind giving this a more descriptive name than "limited"? For example, "mobile_friendly". Thanks!

**Comment by moflo** [comment]
Sure, good point. Made the name change per your feedback.

File: Gruntfile.coffee [hunk 1]
```diff
@@ -72,6 +72,16 @@ module.exports = (grunt) ->
             'bower_components/backbone/backbone.js'
             'bower_components/backbone-deep-model/src/deep-model.js'
           ]
+      limited:
```
Comment (al3x) [inline Gruntfile.coffee]: change to `mobile_friendly`

File: Gruntfile.coffee [hunk 2]
```diff
@@ -72,6 +72,16 @@ module.exports = (grunt) ->
...
+        files:
+          '<%= distFolder %>/formbuilder.js': '<%= compiledFolder %>/*.js'
+          '<%= vendorFolder %>/js/vendor_limited.js': [
```
Comment (al3x) [inline Gruntfile.coffee]: change to `vendor_mobile_friendly.js`

File: Gruntfile.coffee [hunk 3]
```diff
@@ -109,5 +119,6 @@ module.exports = (grunt) ->
 
 
   grunt.registerTask 'default', ALL_TASKS
+  grunt.registerTask 'limited', ['jst:all', 'coffee:all', 'concat:limited', 'stylus:all', 'clean:compiled']
```
Comment (al3x) [inline Gruntfile.coffee]: change to `mobile_friendly`

File: mobile_friendly.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  </style>
+</head>
+<body>
+  <div class='fb-main'></div>
+
+<script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.5.2/underscore-min.js"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.1.0/backbone-min.js"></script>
+<script src="vendor/js/vendor_limited.js"></script>
```
Comment (al3x) [inline mobile_friendly.html]: change to `vendor_mobile_friendly.js`

**Comment by moflo** [comment]
Sorry... updated the Grunt build too.

**Pull request merged**